packages/style-engine/class-wp-style-engine-css-rule.php

```diff
@@ -117,6 +117,11 @@ public function get_css( $should_prettify = false, $indent_count = 0 ) {
 		$declarations_indent = $should_prettify ? $indent_count + 1 : 0;
 		$new_line            = $should_prettify ? "\n" : '';
 		$space               = $should_prettify ? ' ' : '';
-		return $rule_indent . $this->get_selector() . "{$space}{{$new_line}" . $this->declarations->get_declarations_string( $should_prettify, $declarations_indent ) . "{$new_line}{$rule_indent}}";
+
+		$styles = $this->declarations->get_declarations_string();
+		if ( empty( $styles ) ) {
+			return '';
+		}
```

Comment:
No need to print style if there are no declarations. Previously we had things like `.something {}` which doesn't make any sense.

Reply:
Sorry, I merged a fix for it this morning in https://github.com/WordPress/gutenberg/pull/43003 without seeing this first. 

Thanks for spotting that! 🍺